src/FakeItEasy/CallbackConfigurationExtensions.cs

```diff
@@ -10,112 +10,22 @@ namespace FakeItEasy
     /// <summary>
     /// Provides extension methods for <see cref="ICallbackConfiguration{TFake}"/>.
     /// </summary>
-    public static class CallbackConfigurationExtensions
+    public static partial class CallbackConfigurationExtensions
     {
         private const string NameOfInvokesFeature = "invokes";
 
         /// <summary>
         /// Executes the specified action when a matching call is being made. This overload can also be used to fake calls with arguments when they don't need to be accessed.
         /// </summary>
-        /// <typeparam name="TFake">The type of fake object.</typeparam>
+        /// <typeparam name="TInterface">The type of configuration interface to return.</typeparam>

```

Comment:
The type parameter name and description was incorrect and misleading. I changed it to be the same as for similar methods.

Reply:
Nice catch. I think the `returns` value is still a lie. And in the `.tt` too.

Reply:
Oops. I knew I was forgetting something